tailor-packs/java-poc/tailor/models/JdkManual.qll

```diff
@@ -4,7 +4,10 @@ private import semmle.code.java.dataflow.ExternalFlow
 private class JdkManual extends SummaryModelCsv {
   override predicate row(string row) {
     row = [
-      "java.lang;String;true;replace;;;Argument[-1];ReturnValue;taint;manual"
+      "java.lang;String;true;replace;;;Argument[-1];ReturnValue;taint;manual",
+      // Exception's / Throwable's
+      "java.lang;Throwable;true;toString;;;Argument[-1];ReturnValue;taint;manual",
```

Comment:
This line currently doesn't seem to have any effect. I don't get an additional result due to it.